### PR TITLE
Updated Readme.md

### PR DESCRIPTION
**Pull request by aniketsingh98571**
Changed the frontend and backend steps in the Readme.md file.

## Issue that this pull request solves
#74 
 Closes: # (issue number)

## Proposed changes
Changed frontend and backend steps in Readme.md
## Types of changes

_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] My changes does not break the current system and it passes all the current test cases.

## Screenshots

Please attach the screenshots of the changes made in case of change in user interface

## Other information

Any other information that is important to this pull request

### CODE REVIEW

**Comment by aniketsingh98571** [comment]
@Kajol-Kumari  Please look into it. Reference #74 .Thank You.

**Comment by jackfrost13** [comment]
@aniketsingh98571 Hi, the changes looks a bit disproportionate. Can you add a getting started section before Frontend where you show the steps to fork ==> clone user repo and cd repo
For the front-end and backend steps should start with cd frontend and cd backend respectively

**Comment by aniketsingh98571** [comment]
@jackfrost13 Sure I will add `Getting Started` before frontend and also regarding the concept of directories. How directly can we switch to `cd frontend`, firstly after cloning the repo we should first `cd Community-Website` and then `cd frontend`, I have shown all the steps precisely because it will help beginners also to work on the repo because the person, who don't have any knowledge of reactjs and nodejs how will they know how to install all dependencies to their machine. in a directory. This is the reason.

**Comment by pushkarkhadase** [comment]
As you are using nodemon instead of doing npm run dev why dont you do npm start? as it will also start the server into the developement mode.

**Comment by jackfrost13** [comment]
> As you are using nodemon instead of doing npm run dev why dont you do npm start? as it will also start the server into the developement mode.

No, it will not start the server in dev mode automatically.
`npm start` and `npm run dev` both are configured scripts. None of them has a predefined effect on the server.
`npm start` is mostly used to run the server and `npm run dev` for development mode. We have also configured it in the same way. I hope this answer satisfies you. You can look at the `package.json` file where you can find the scripts

**Comment by jackfrost13** [comment]
> @jackfrost13 Sure I will add `Getting Started` before frontend and also regarding the concept of directories. How directly can we switch to `cd frontend`, firstly after cloning the repo we should first `cd Community-Website` and then `cd frontend`, I have shown all the steps precisely because it will help beginners also to work on the repo because the person, who don't have any knowledge of reactjs and nodejs how will they know how to install all dependencies to their machine. in a directory. This is the reason.

Yes,  `cd Community-Website` should be in the getting started section. 
`cd frontend` and `cd backend` in their sections respectively.

**Review by jackfrost13 — CHANGES_REQUESTED** [review]
Refer to previous comments https://github.com/HITK-TECH-Community/Community-Website/pull/95#issuecomment-751635662.

If we have git clone in both frontend and backend section, it will be somewhat misleading to the beginners whether they need to clone it again or not. That's why those should go into the getting started section

**Review by aniketsingh98571 — COMMENTED** [review]
Ok Go ahead.

**Comment by aniketsingh98571** [comment]
Ok now I got it @jackfrost13, then do one thing before frontend adds `git clone` --->` cd Community-Website` and then go for particular sections.

**Comment by jackfrost13** [comment]
@aniketsingh98571 Yes please go ahead with the changes and once you are done, you can re-request for my review

**Comment by aniketsingh98571** [comment]
@jackfrost13  Now look into the new updated file, and check does it suits what you are telling of?
@Kajol-Kumari  Please review it.

File: README.md [hunk 1]
```diff
@@ -16,29 +16,32 @@ This project aims to build the official site for the community page so that any
 
 ## Tech Stack 🗃
 
- <img src="https://img.shields.io/badge/-MongoDB-yellow?style=flat&logo=mongoDB"> <img src="https://img.shields.io/badge/-expressJS-yellowgreen"> <img src="https://img.shields.io/badge/-ReactJS-%2300bfff%20">  <img src="https://img.shields.io/badge/-NodeJS-black?style=flat&logo=node.js">  
+ <img src="https://img.shields.io/badge/-MongoDB-yellow?style=flat&logo=mongoDB"> <img src="https://img.shields.io/badge/-expressJS-yellowgreen"> <img src="https://img.shields.io/badge/-ReactJS-%2300bfff%20">  <img src="https://img.shields.io/badge/-NodeJS-black?style=flat&logo=node.js"> 
+ 
+##  Getting Started 👨‍💻
+
+1. `git clone`
```
Comment (jackfrost13) [inline README.md:23]: git clone is not the right action to perform for getting started
One should fork the repository first.
Please make the changes accordingly.

**Comment by aniketsingh98571** [comment]
@jackfrost13  Requested for review...

File: README.md [hunk 2]
```diff
@@ -16,29 +16,32 @@ This project aims to build the official site for the community page so that any
...
 Following are the steps to run the frontend of the community-website.All the code will go in `frontend` directory. 
 
-1. `git clone`
-2. Navigate to `Community-Website` folder using `cd Community-Website`
-3. `cd frontend`
-4. Run `npm install`. It will install all the required packages and dependencies.
-5. Run `npm start` to run the server.
-6. Navigate to [http://localhost:3000](http://localhost:3000) to view it in the browser.
+
+1. `cd frontend`
```
Comment (jackfrost13) [inline README.md:31]: Please be a little more descriptive like the backend section.

**Review by jackfrost13 — CHANGES_REQUESTED** [review]

**Comment by jackfrost13** [comment]
@aniketsingh98571 I have reviewed it.
Please go through the comments, try to be a little more descriptive `git clone` doesn't seem to be a complete command. Please update it accordingly

File: README.md (hunk 1 above)
Comment (aniketsingh98571) [reply README.md:23]: then after forking can I add `git clone`

File: README.md (hunk 2 above)
Comment (aniketsingh98571) [reply README.md:31]: but in the heading, it was mentioned...

File: README.md (hunk 1 above)
Comment (jackfrost13) [reply README.md:23]: Yes, but `git clone` is not a complete command.
Please update it accordingly

File: README.md (hunk 2 above)
Comment (jackfrost13) [reply README.md:31]: I didn't get your point. Please elaborate!

File: README.md (hunk 1 above)
Comment (aniketsingh98571) [reply README.md:23]: okay

File: README.md (hunk 2 above)
Comment (aniketsingh98571) [reply README.md:31]: Means every section has its own heading like 'frontend', 'backend', so how can I be more descriptive.

**Comment by aniketsingh98571** [comment]
@jackfrost13 Check it now.

File: README.md [hunk 3]
```diff
@@ -19,16 +19,18 @@ This project aims to build the official site for the community page so that any
  <img src="https://img.shields.io/badge/-MongoDB-yellow?style=flat&logo=mongoDB"> <img src="https://img.shields.io/badge/-expressJS-yellowgreen"> <img src="https://img.shields.io/badge/-ReactJS-%2300bfff%20">  <img src="https://img.shields.io/badge/-NodeJS-black?style=flat&logo=node.js"> 
  
 ##  Getting Started 👨‍💻
+### Setup the repository to your local environment.
 
-1. `git clone`
-2. `cd Community-Website`
+1. `fork` the repository ,     Creates a replica of repository to your local environment.
+2. `git clone https://github.com/HITK-TECH-Community/Community-Website.git`   Download all files to your machine.
```
Comment (jackfrost13) [inline README.md:25]: one must not clone this repository.
The forked repo must be cloned

File: README.md [hunk 4]
```diff
@@ -16,13 +16,21 @@ This project aims to build the official site for the community page so that any
...
+3. `cd Community-Website`  move to the root directory.
 
 ## Frontend ⚛️
 
-Following are the steps to run the frontend of the community-website.
+Following are the steps to run the frontend of the community-website.All the code will go in `frontend` directory. 
+
 
-1. Navigate to `community-website` folder using `cd community-website`
+1. `cd frontend` Naviagate to `frontend` section.
```
Comment (jackfrost13) [inline README.md:33]: You need to rephrase this like it was done in the backend section

**Review by jackfrost13 — CHANGES_REQUESTED** [review]

**Comment by aniketsingh98571** [comment]
@jackfrost13 Now check it.

**Comment by jackfrost13** [comment]
you still to rectify this https://github.com/HITK-TECH-Community/Community-Website/pull/95#discussion_r549356647

**Comment by aniketsingh98571** [comment]
@jackfrost13  are you a mentor/PA for this project.?

**Comment by jackfrost13** [comment]
> @jackfrost13 are you a mentor/PA for this project.?

Yes

**Comment by aniketsingh98571** [comment]
for SWOC?

**Comment by jackfrost13** [comment]
> for SWOC?

We don't have separate mentors for any of the events. Mentors are for the project

**Comment by aniketsingh98571** [comment]
Now, are you talking about the line below the Frontend headline? That I need to change?

**Comment by jackfrost13** [comment]
> Now, are you talking about the line below the Frontend headline? That I need to change?

No, the git clone command is incorrect.
Refer https://github.com/HITK-TECH-Community/Community-Website/pull/95#discussion_r549356647

**Comment by aniketsingh98571** [comment]
@jackfrost13 
`git clone https://github.com/YOUR-USERNAME/Community-Website`
are you talking about this?

**Comment by jackfrost13** [comment]
> @jackfrost13
> `git clone https://github.com/YOUR-USERNAME/Community-Website`
> are you talking about this?

Yes. Please update this and it's good to go :)

**Comment by aniketsingh98571** [comment]
@jackfrost13 Done.

**Review by jackfrost13 — APPROVED** [review]

**Review by jackfrost13 — APPROVED** [review]

**Comment by jackfrost13** [comment]
Closes #74

**Comment by Kajol-Kumari** [comment]
@all-contributors please add @aniketsingh98571  for doc

**Pull request merged**